app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -631,7 +635,8 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
       val attributes = Array(labelCounter("CurbRamp"), labelCounter("NoCurbRamp"), labelCounter("Obstacle"), labelCounter("SurfaceProblem")).map(_.toDouble)
       val significance = Array(0.75, -1.0, -1.0, -1.0)
       val accessScore: Double = computeAccessScore(attributes, significance)
-      AccessScoreStreet(edge, accessScore, attributes, significance)
+      val osmId = StreetEdgeTable.osmStreetIds(edge.streetEdgeId);
```

Comment:
I agree with not adding the OSM ID to the `StreetEdge` class like you had for the `GlobalAttributeWithLabelForAPI` and `GlobalAttributeForAPI`. But I think that we should modify this slightly so that we only run one query on all the streets instead of running a query to get the `osm_way_id` for each street individually.

This entail modifying your query to take in a `List`/`Seq`/`Set` of `Int`s instead of just one. Then either calling that outside of this loop, or outside of this function altogether (taking in that list of IDs as a parameter to this function).

On this note, I'll add a comment to the `osmStreetIds()` function about the syntax there :)